packages/website/components/SidebarSection.tsx

```diff
@@ -98,6 +98,16 @@ export function SidebarSection({
   const { currentPage } = useCurrentLocation();
   const { data: session } = useSession();
 
+  // move the deprecated link section to the end of the components list
+  const deprecatedLinkSectionIndex = links.findIndex((link) =>
+    link.title.includes('Deprecated V3 Components'),
+  );
+
+  if (deprecatedLinkSectionIndex !== -1) {
+    const removedElement = links.splice(deprecatedLinkSectionIndex, 1);
+    links.push(removedElement[0]);
+  }
```

Comment:
💭 wouldn't the side bar look much better if the sections are collapsed but the one the current active component is in, expanded?

Reply:
I think the sidebar UX could be improved, the mix of collapsable/static elements, font weights, and indentation levels (subsection links) makes it hard to scan it IMO.